qa/hooks/commit-msg

```diff
@@ -4,7 +4,7 @@ COMMIT_MESSAGE="$(cat $MSG_FILE)"
 REGEX='^(Close|Closes|Closed|Fix|Fixes|Fixed|Resolve|Resolves|Resolved) #([0-9]+)'
 if [[ $COMMIT_MESSAGE =~ $REGEX ]]; then
  TICKET=${BASH_REMATCH[2]}
- RESULT=$(curl -s https://api.github.com/repos/kadai-iokadai/issues/$TICKET)
+ RESULT=$(curl -s https://api.github.com/repos/kadai-io/kadai/issues/$TICKET)
```

Comment:
I think that I solved this one already, can you try to rebase your branch on kadai master?

Reply:
No it is still not yet solved